SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriverGameplayDebuggerContext.cpp

```diff
@@ -242,6 +255,84 @@ void USpatialNetDriverGameplayDebuggerContext::TickServer()
 
 		It.RemoveCurrent();
 	}
+
+	for (auto& Pair : TrackedEntities)
+	{
+		if (Pair.Value.Component.TrackPlayer)
+		{
+			if (!NetDriver->HasServerAuthority(Pair.Key))
+			{
+				// Do not track player if we don't have authority
+				continue;
+			}
+
+			TWeakObjectPtr<AGameplayDebuggerCategoryReplicator> ReplicatorWeakObjectPtr = Pair.Value.ReplicatorWeakObjectPtr;
+			AGameplayDebuggerCategoryReplicator* Replicator = ReplicatorWeakObjectPtr.Get();
+			if (Replicator == nullptr)
+			{
+				// Not unexpected - assume latency and wait for actor to appear
+				continue;
+			}
+
+			const AActor* PlayerControllerActor = Cast<AActor>(Replicator->GetReplicationOwner());
+			if (!ensureMsgf(PlayerControllerActor != nullptr, TEXT("Serious error: Cannot find player controller for replicated gameplay debugger actor")))
+			{
+				continue;
+			}
+
+			check(NetDriver->LoadBalanceStrategy);
+			auto GetActorVirtualWorkerId = [](const AActor& InActor, const USpatialNetDriver& InNetDriver, const SpatialGDK::FSubView& InSubView)->VirtualWorkerId
```

Comment:
Should this be a private method? I understand it is only used here, but we're already two indentation levels in at this point and it might read easier if we don't go even deeper.